feature-staking-api/src/main/java/jp/co/soramitsu/feature_staking_api/domain/api/StakingRepository.kt

```diff
@@ -64,7 +66,17 @@ suspend fun StakingRepository.historicalEras(): List<BigInteger> {
     val currentEra = getCurrentEraIndex().toInt()
     val historyDepth = getHistoryDepth().toInt()
 
-    val historicalRange = (currentEra - historyDepth) until activeEra
+    val historicalRange = (currentEra - historyDepth)..activeEra
 
     return historicalRange.map(Int::toBigInteger)
 }
+
+suspend fun StakingRepository.payoutCheckedEras(): List<BigInteger> {
+    val activeEra = getActiveEraIndex().toInt()
+    val currentEra = getCurrentEraIndex().toInt()
+    val historyDepth = getHistoryDepth().toInt()
+
+    val historicalRange = (currentEra - historyDepth) until activeEra
+
+    return historicalRange.map(kotlin.Int::toBigInteger)
+}
```

Comment:
There seems to be a lot of duplication with the method above. Can we refactor it?

Reply:
Done